server/plugin/command.go

```diff
@@ -137,10 +137,12 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 				return &model.CommandResponse{}, nil
 			}
 
-			if _, err := client.ActivateSubscriptions(); err != nil {
-				p.API.LogError("Unable to check or activate subscriptions in ServiceNow.", "Error", err.Error())
-				p.postCommandResponse(args, p.handleClientError(nil, nil, err, isSysAdmin, 0, args.UserId, ""))
-				return &model.CommandResponse{}, nil
+			if action != constants.CommandSearchAndShare {
```

Comment:
we can use the `==` condition here as we then don't have to add more conditions when more slash commands will be added.